tests/libyui/menu_hotkeys_1177760_spec.rb

```diff
@@ -0,0 +1,36 @@
+require_relative "rspec_tmux_tui"
+
+describe "Menu Item" do
+  bug = "1177760"
+  around(:each) do |ex|
+    @base = "menu_hotkeys_#{bug}"
```

Comment:
where is it used?

Reply:
in the screenshot filenames: `@tui.capture_pane_to("#{@base}-1-initial")`